### ISSUE TITLE
overflow-y property in .panel class breaks the tooltip

### ISSUE THREAD

**Issue by cyonder** [picturepan2/spectre#426]
<img width="416" alt="screen shot 2018-03-31 at 4 25 44 am" src="https://user-images.githubusercontent.com/10395746/38161371-b5e539b2-349b-11e8-8ba6-3a898cd4490e.png">

```
.panel .panel-body {
    -ms-flex: 1 1 auto;
    flex: 1 1 auto;
    /* overflow-y: auto; */
    padding: 0 .8rem;
}
```

**Comment by picturepan2** [comment]
@cyonder Yes, but removing `overflow-y: auto;` makes it unscrollable. :( Any idea to fix it?

**Comment by cyonder** [comment]
Unfortunately I wasn't able to fix it. That's why I am using `tooltip-left` in the button class.
